keps/sig-cloud-provider/2436-controller-manager-leader-migration/README.md

```diff
@@ -178,16 +178,17 @@ type ControllerLeaderConfiguration struct {
 
 	// Component is the name of the component in which the controller will be running.
 	// E.g. kube-controller-manager, cloud-controller-manager, etc
+	// Or '*' meaning the controller can be run under any component that reads the configuration
```

Comment:
Slightly pedantic but the controller will be run in any component with migration enabled.